### ISSUE TITLE
Any advice for scaling up to clip-G/bigG?

### ISSUE THREAD

**Issue by jiashenggu** [beichenzbc/Long-CLIP#18]
I reproduced your long clip-L. When I try to train long clip-G, at the end of the training, the test average for shareGPT4V is about 99, but the performance of flickr30k drops.

**Comment by beichenzbc** [comment]
Due to limited computational resource, we can't give a precise solution. Here is some advice based on our experience:
1. a smaller learning rate. 
2. a larger batch size
3. a higher ratio on the loss of short caption

**Comment by jiashenggu** [comment]
> Due to limited computational resource, we can't give a precise solution. Here is some advice based on our experience:
> 1. a smaller learning rate. 
> 2. a larger batch size
> 3. a higher ratio on the loss of short caption

How about using a value larger than 20 as the short position embedding? I'm confused by this because when evaluating 1000 sharegptv examples, the results are clearly better if the first 30 markers are used.

**Comment by beichenzbc** [comment]
Positional embedding interpolation will affect the performance more or less. It's a trade-off between maximum length supported and performance. You may try different value according to your need.
